README.md

```diff
@@ -33,8 +33,9 @@ Installing `MSS` from the `conda-forge` channel can be achieved by adding `conda
 
 Once the `conda-forge` channel has been enabled, `mss` can be installed with:
 
-    $ conda create -n mssenv mamba
+    $ conda create -n mssenv python=3.9
```

Comment:
please remove =3.9

seems there are more occurencies of this problem interpreting a version string as float number ;)

adding the python modul will release automatically the fixation and we will not need to adjust when we can use 3.10 agaon

Reply:
and if conda "create -n mssenv mamba python" works we should do that